packages/mobile/ios/celo.xcodeproj/project.pbxproj

```diff
@@ -8,6 +8,9 @@
 
 /* Begin PBXBuildFile section */
 		00E356F31AD99517003FC87E /* celoTests.m in Sources */ = {isa = PBXBuildFile; fileRef = 00E356F21AD99517003FC87E /* celoTests.m */; };
+		0317E786245B696200AAAC2E /* Inter-Regular.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 0317E783245B696200AAAC2E /* Inter-Regular.ttf */; };
```

Comment:
Comment about /mobile/fonts/Inter-*.ttf, not this file:
Do we need the font files placed there AND in android folders AND in ios folders?

Reply:
yeah this is something I am wondering too.

Reply:
I managed to fix so that there the ios build gets fonts directly from main fonts folder. 

android sounds like to will need some other build steps. Id like to have that be a separate ticket.